### PR TITLE
DO-241: Safari issue fixed.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/common/PictureViewer/_PictureViewer.scss [hunk 1]
```diff
@@ -122,6 +134,16 @@
   }
 }
 
+.member-profile-picture-viewer__list-page .image-gallery-thumbnails-wrapper {
```
Comment (mohitsaladiya) [inline src/components/common/PictureViewer/_PictureViewer.scss:137]: Why are we using .member-profile-picture-viewer__list-page in _PictureViewer.scss? It should be under _MemberProfilePictureViewer.scss

File: src/components/common/PictureViewer/_PictureViewer.scss [hunk 2]
```diff
@@ -63,15 +63,27 @@
...
 .image-gallery-content .image-gallery-slide .image-gallery-image {
   max-height: calc(100vh - 300px);
   outline: none;
   @media (max-width: 767px) {
-    max-height: calc(100vh - 230px);
+    max-height: calc(100vh - 280px) !important;
+  }
+}
+
+.member-profile-picture-viewer__list-page .image-gallery-content .image-gallery-slide .image-gallery-image {
```
Comment (mohitsaladiya) [inline src/components/common/PictureViewer/_PictureViewer.scss:82]: Why are we using .member-profile-picture-viewer__list-page in _PictureViewer.scss? It should be under _MemberProfilePictureViewer.scss

File: src/components/common/PictureViewer/_PictureViewer.scss (hunk 1 above)
Comment (amitjain24524) [reply src/components/common/PictureViewer/_PictureViewer.scss]: done

File: src/components/common/PictureViewer/_PictureViewer.scss (hunk 2 above)
Comment (amitjain24524) [reply src/components/common/PictureViewer/_PictureViewer.scss]: done

**Pull request merged**